src/main/java/by/custom/utilcalculator/service/BotFieldsManager.java

```diff
@@ -0,0 +1,100 @@
+package by.custom.utilcalculator.service;
+
+import by.custom.utilcalculator.directory.BotEntity;
+import by.custom.utilcalculator.directory.resources.Commands;
+import by.custom.utilcalculator.directory.steps.*;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+
+public class BotFieldsManager {
+    private final BotEntity botEntity;
+    private final MessagesManager messagesManager;
+
+    public BotFieldsManager() {
+        botEntity = BotEntity.getInstance();
+        messagesManager = MessagesManager.getInstance();
+    }
+
+    public static BotFieldsManager getInstance() {
+        return BotFieldsManagerHolder.BOT_FIELDS_MANAGER;
+    }
+
+    public SendMessage setCarOriginFieldAndGetNextMessageForUser(Message inputMessage) {
```

Comment:
I think in this case we can use a simpler but enough informative method name. The same for other methods.
```suggestion
    public String processCarOrigin(String inputMessage) {
```

Reply:
Fixed. 
There are two operations in this methods. First operation - to change the field of Bot object. 
`case Command.EAES -> botEntity.setCountryOrigin(CountryOrigin.EAES);`
The second operation is to return String with message to send user like an answer.
`return messagesManager.createSendMessage();`
Is it OK or we must to split this method in two different methods?

Reply:
It's ok for service. It's a right place to aggregate these things, store progress, and prepare answer.